Dockerfile

```diff
@@ -3,6 +3,9 @@ ARG NODE_ENV=production
 ARG REACT_APP_TITLE
 ARG REACT_APP_SUBTITLE
 ARG REACT_APP_INCIDENT_MAP
+ARG REACT_APP_OPENWEATHER
+ARG REACT_APP_API_URL 
+ARG REACT_APP_ICON_URL 
```

Comment:
You should consider making all of these more descriptive. For example, think about how you would understand `REACT_APP_API_URL` when you come back to it in a month. Which API?